stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -331,6 +333,66 @@ private void getStyleFromFloat(
     styles.store(name, String.valueOf(value), isDefaultValue(value));
   }
 
+  private void getStyleFromObject(
+      View view,
+      String name,
+      Object value,
+      @Nullable ViewDebug.ExportedProperty annotation,
+      StyleAccumulator styles) {
+    if (annotation != null && annotation.deepExport() && value != null) {
+      Field[] fields = value.getClass().getFields();
+
+      Object propertyValue;
+      for (Field field : fields) {
+        try {
+            field.setAccessible(true);
+            propertyValue = field.get(value);
+        } catch (IllegalAccessException e) {
+          LogUtil.e(
+              e,
+              "failed to get property of name: \"" + name + "\" of object: " + String.valueOf(value));
+          propertyValue = null;
```

Comment:
We can just 'return' here instead of setting `propertyValue` to null. And then we can remove the `if (propertyValue != null)` down below